pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -64,6 +65,8 @@ import (
 var (
 	// LocalCertDir replaces the "cert-chain", "signing-cert" and "signing-key" flags in citadel - Istio installer is
 	// requires a secret named "cacerts" with specific files inside.
+	// When Custom CA Addr is defined, LocalCertDir also contains 'custom-ca.crt', 'custom-ca.key' and 'root-cert.pem'
```

Comment:
Let's keep those in a separate secret. I don't think we'll even have this directory or files if an external CA is used - why would we generate self-signed root CA if using an external CA ?

Reply:
Also for new files - please use ./var/lib/istio/something...